api-server/src/server/middlewares/error-handlers.js

```diff
@@ -48,22 +49,16 @@ export default function prodErrorHandler() {
       console.error(errTemplate(err, req));
     }
 
-    if (type === 'html') {
-      if (typeof req.flash === 'function') {
-        req.flash(handled.type || 'danger', message);
-      }
-      return res.redirectWithFlash(redirectTo);
-      // json
-    } else if (type === 'json') {
-      res.setHeader('Content-Type', 'application/json');
-      return res.send({
+    if (type === 'json') {
+      return res.json({
         type: handled.type || 'errors',
         message
       });
-      // plain text
     } else {
-      res.setHeader('Content-Type', 'text/plain');
-      return res.send(message);
```

Comment:
Not sure if this was intended, but text type will be handled similar to html with this change.